### ISSUE TITLE
Can we add a footer?

### ISSUE THREAD

**Issue by FottyM** [FottyM/mutunda#8]
The technologies are clipped at the bottom, it would be nice to have a footer.

**Comment by george-misan** [comment]
Yes, I will add it his week, I am looking for suitable icons for it.

**Comment by FottyM** [comment]
Cool, cool, thanks.

**Comment by FottyM** [comment]
Once you've created a PR, attach it to his issue so that we can close it.
